config.go

```diff
@@ -120,12 +119,8 @@ var (
 
 func init() {
 	AppPath, _ = filepath.Abs(filepath.Dir(os.Args[0]))
-	workPath, _ = os.Getwd()
```

Comment:
I think the change at this position is OK. However, after starting, should beego app change current dir to AppPath? We should make a explicit hypothesis - relative path settings is relative to AppPath or current path - this decide that beego should change current dir to AppPath or not.